modules/apps/foundation/frontend-theme/theme3/src/js/main.js

```diff
@@ -0,0 +1,75 @@
+AUI().ready(
+
+	/*
+	This function gets loaded when all the HTML, not including the portlets, is
+	loaded.
+	*/
+function() {
+
+
+}
+);
+Liferay.Portlet.ready(
+
+	/*
+	This function gets loaded after each and every portlet on the page.
+
+	portletId: the current portlet's id
+	node: the Alloy Node object of the current portlet
+	*/
+
+	function(portletId, node) {
+	}
+);
+
+Liferay.on(
+	'allPortletsReady',
+
+	
+
+	function() {
+
+	
+	}
+);
+
+
+$(document).ready(function () {
```

Comment:
Since you are using jQuery, you should leverage the one that is already available in Liferay using ```AUI.$(...);```.